src/dmd/utils.d

```diff
@@ -19,6 +19,7 @@ import dmd.root.filename;
 import dmd.common.outbuffer;
 import dmd.root.string;
 
+nothrow:
```

Comment:
?

Reply:
Because I was trying to make preprocess() nothrow. I failed due to other problems, but the nothrows should still stick where they can.

Reply:
They should go in a different PR then.